app/components/button_locale/ButtonLocale.tsx

```diff
@@ -0,0 +1,18 @@
+import nextI18nextConfig from "../../../next-i18next.config"
+import LanguagesToogle from "../language_toogle/LanguagesToogle"
+
+interface ButtonLocaleProps {
```

Comment:
could we have this interface on an external file?

Reply:
yes, it was resolved